internal/cmd/base/base.go

```diff
@@ -68,6 +68,7 @@ const (
 	EnvKeyringType     = "BOUNDARY_KEYRING_TYPE"
 	envRecoveryConfig  = "BOUNDARY_RECOVERY_CONFIG"
 	envSkipCacheDaemon = "BOUNDARY_SKIP_CACHE_DAEMON"
+	envSkipFerry       = "BOUNDARY_SKIP_FERRY"
```

Comment:
Should we add _DAEMON for consistency?
```suggestion
	envSkipFerry       = "BOUNDARY_SKIP_FERRY_DAEMON"
```